File: Praktikum 7/Python/PRAK603-2310817210004-Allano Lintang Ertantora.py
satu, dua = map(int, input().split())
if satu != dua:
    print("Jumlah tidak sama")
else:
    barisSatu = list(map(int, input().split()))
    barisDua = list(map(int, input().split()))
    for i in range(satu):
        print(barisSatu[i] * barisDua[i], end=" ")

File: Praktikum 7/C/PRAK603-2310817210004-Allano-Lintang-Ertantora.c
#include <stdio.h>
int main(){
    int satu, dua;
    scanf("%d %d", &satu, &dua);
    int barisSatu[satu], barisDua[dua];
    if (satu != dua){
        printf("Jumlah tidak sama\n");
    }else{
        for(int i = 0; i < satu; i++){
            scanf("%d", &barisSatu[i]);
        }
        for(int j = 0; j < dua; j++){
            scanf("%d", &barisDua[j]);
        }
        for(int k = 0; k < satu; k++){
            printf("%d ", barisSatu[k] * barisDua[k]);
        }
    }
    return 0;
}